MenMa WatchKit Extension/InterfaceController.swift

```diff
@@ -58,11 +58,13 @@ class InterfaceController: WKInterfaceController, WCSessionDelegate {
         
         let messageToSend = ["value": "message sent successfully"]
         session.sendMessage(messageToSend, replyHandler: { replyMessage in
-            //            //handle and present the message on screen
-            //            let value = replyMessage["value"] as? Venue
-            //            //            self.messageLabel.setText(value)
+            //handle and present the message on screen
+            let value = replyMessage["value"] as? Venue
```

Comment:
> In my local feature/filter_venue build I don't have anything valuable commented out.

Sorry, what do you mean this? You mean that `let value = replyMessage["value"] as? Venue` is not commented out in your code? I think [this commit](https://github.com/keitaito/MenMa/commit/179f0ea71fc788ca067f195ce8f1a690e785b480) commented out the code. Can I ask you to check if your branch is up-to-date?